eng/Signing.props

```diff
@@ -1,5 +1,14 @@
 <Project>
 
+  <PropertyGroup>
+    <Product>Sdk</Product>
+    <BlobStoragePartialRelativePath>$(Product)</BlobStoragePartialRelativePath>
+    <BlobStoragePartialRelativePath Condition="'$(IsNotOrchestratedPublish)' == 'false'">assets/$(Product)</BlobStoragePartialRelativePath>
+    <BlobStorageVersion>$([System.IO.File]::ReadAllText('$(ArtifactsTmpDir)FullNugetVersion.version'))</BlobStorageVersion>
+    <BlobStoragePartialRelativePath>$(BlobStoragePartialRelativePath)/$(BlobStorageVersion)</BlobStoragePartialRelativePath>
+    <PublishBinariesAndBadge Condition=" '$(PublishBinariesAndBadge)' == '' ">true</PublishBinariesAndBadge>
+  </PropertyGroup>
+
   <ItemGroup Condition="'$(PostBuildSign)' != 'true'">
```

Comment:
I think this line should get removed. I think that `AllowEmptySignList` and `AllowEmptySignPostBuildList` should also get removed now.